### PR TITLE
refactor: update formats-common

### PR DESCRIPTION
**Pull request by tpluscode**
Also, replaces the direct use of `@zazuko/env`

### CODE REVIEW

File: packages/rdf/test/append.test.js [hunk 1]
```diff
@@ -150,7 +147,7 @@ describe('metadata.append', () => {
     }, Error, /unknown protocol/)
   })
 
-  it('fails at file not found', async () => {
+  it.skip('fails at file not found', async () => {
```
Comment (tpluscode) [inline packages/rdf/test/append.test.js:150]: I'm willing to risk disabling this and the other test. An error is indeed thrown but somehow the stream+async no longer plays nice with mocha. Does not seem like a great deal (?)

Comment (ludovicm67) [reply packages/rdf/test/append.test.js:150]: And by using a pattern like this:

```js
try {
  await append()
  throw new Error('Error was not thrown as expected')
} catch (error) {
  // Assert the error
  strictEqual(error.message, 'unknown protocol')
}
```

?

**Review by ludovicm67 — APPROVED** [review]
LGTM ; maybe creating an issue for investigating what is wrong with the test, to be sure to not forget about it?

File: packages/rdf/test/append.test.js (hunk 1 above)
Comment (tpluscode) [reply packages/rdf/test/append.test.js:150]: No, I tried multiple ways. Somehow, an error in `localFetch` is not being caught from the stream and reaches global mocha handler.

Comment (ludovicm67) [reply packages/rdf/test/append.test.js:150]: Oh ok… this is quite a strange situation :/

**Pull request merged**